MAP-LIB_ProjectRules.md

```diff
@@ -276,9 +276,9 @@ Members:
   | `Mechanics.MultiBody`        | Martin Otter, Jakub Tobolar                                     |
   | `Mechanics.Rotational`       | Anton Haumer, Christian Kral, Martin Otter, Jakub Tobolar       |
   | `Mechanics.Translational`    | Anton Haumer, Christian Kral, Martin Otter, Jakub Tobolar       |
-  | `Fluid`                      | Francesco Casella, Rüdiger Franke, Hubertus Tummescheit         |
-  | `Fluid.Dissipation`          | Francesco Casella, Stefan Wischhusen                            |
-  | `Media`                      | Francesco Casella, Rüdiger Franke, Hubertus Tummescheit         |
+  | `Fluid`                      | Francesco Casella, Arunkumar Narashiman                         |
```

Comment:
This violates 5.1.2.2.b

> New Library Officers can be assigned by the Project Members for the same library/sublibrary, with a [qualified majority](https://github.com/modelica/MAP-LIB_ProjectRules/blob/master/MAP-LIB_ProjectRules.md#52-voting) of all Project Members.

Same for `Media` two lines below.

Reply:
@beutlich we already had this decision in a previous meeting. We just need to get this recorded